civictechprojects/static/css/DiscoverPageDescription.css

```diff
@@ -0,0 +1,4 @@
+.DiscoverPageDescription-root {
```

Comment:
Is this used?

Reply:
No, I think we'll need it only if we'll have long description with more than one paragraph.